client/my-sites/plugins/plugins-manage-mixin.js

```diff
@@ -7,14 +7,10 @@ module.exports = {
 		if ( ! selectedSite ) {
 			return true;
 		}
-		if ( ! selectedSite || ! selectedSite.modulesFetched ) {
-			return true;
-		}
-		if ( ! selectedSite.getModule( 'manage' ) ||
-				! selectedSite.getModule( 'manage' ).active ) {
+		if ( ! selectedSite.isModuleActive( 'manage' ) ) {
```

Comment:
I think we should also add there another if:
```
if ( ! selectedSite.modules ) {
    return true;
}
```
for the case where we don't know yet if it can manage or not